bin/fuel-core/src/cli/run.rs

```diff
@@ -60,52 +60,59 @@ mod relayer;
 /// Run the Fuel client node locally.
 #[derive(Debug, Clone, Parser)]
 pub struct Command {
-    #[clap(long = "ip", default_value = "127.0.0.1", value_parser)]
+    #[clap(long = "ip", default_value = "127.0.0.1", value_parser, env)]
```

Comment:
Could you specify `env = PORT`, and use all capitals as convention for environment variables?

Reply:
this is IP not port, and the default naming for envs is SCREAMING_SNAKE_CASE:

> env [= <str>]: [Arg::env](https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.env) (needs [env feature](https://docs.rs/clap/latest/clap/_features/index.html) enabled)
When not present: no env set
Without <str>: defaults to the case-converted field name
https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes

> rename_all_env = <string_literal>: Override default field name case conversion for env variables for [Arg::env](https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.env)
When not present: "SCREAMING_SNAKE_CASE"
Available values: "camelCase", "kebab-case", "PascalCase", "SCREAMING_SNAKE_CASE", "snake_case", "lower", "UPPER", "verbatim"
https://docs.rs/clap/latest/clap/_derive/index.html#command-attributes


You can also confirm the casing of the variable names by running `cargo run --bin fuel-core -- run --help`:
```
 fuel-core git:(Voxelot/clap-env) cargo run --bin fuel-core -- run --help
    Finished dev [unoptimized + debuginfo] target(s) in 0.23s
     Running `target/debug/fuel-core run --help`
Run the Fuel client node locally

Usage: fuel-core run [OPTIONS]

Options:
      --ip <IP>
          [env: IP=]
          [default: 127.0.0.1]

      --port <PORT>
          [env: PORT=]
          [default: 4000]

      --db-path <DB_PATH>
          [env: DB_PATH=]
          [default: /Users/voxelot/.fuel/db]

      --db-type <DATABASE_TYPE>
          [env: DATABASE_TYPE=]
          [default: rocks-db]
          [possible values: in-memory, rocks-db]
```

Reply:
Oh this also explains why I was also having trouble testing configuring via env locally